tools/ci/tc/decision.py

```diff
@@ -290,10 +290,15 @@ def add_task(task_name, task):
     # declare a sink task that depends on all the other tasks completing, and
     # checks if they have succeeded. We can then make the sink task the sole
     # required task for GitHub.
-    depends_on_ids = [x[0] for x in task_id_map.values()]
-    sink_task = all_tasks['sink-task']
-    sink_task['command'] += ' {0}'.format(' '.join(depends_on_ids))
-    task_id_map['sink-task'] = create_tc_task(event, sink_task, taskgroup_id, depends_on_ids)
+    is_pr, _ = get_triggers(event)
+    if is_pr:
```

Comment:
We can specify this in the yml using
```
use:
  - trigger-pr
```

and then just make this conditional on the sink task existing.